roles/keycloak/tasks/main.yml

```diff
@@ -75,7 +87,7 @@
     ingress_annotations:
       cert-manager.io/cluster-issuer: atmosphere
 
-- name: Disable pxc strict mode
+- name: Eable pxc strict mode
```

Comment:
```suggestion
- name: Enable pxc strict mode
```

Reply:
how about v3-standard-6?

Reply:
I don't know if we want to add a whole another pool of that flavor, since we're running it inside our cloud, the cost isn't much to worry about that this point :)